superset-frontend/src/components/LastUpdated/index.tsx

```diff
@@ -0,0 +1,80 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React, { useEffect, useState, FunctionComponent } from 'react';
+import moment from 'moment';
+import { t, styled } from '@superset-ui/core';
+import Icon from 'src/components/Icon';
+
+const REFRESH_INTERVAL = 60000; // every minute
+
+interface LastUpdatedProps {
+  updatedAt: string | Date | number;
```

Comment:
huh... I'm surprised we don't have a base `timestampType` for this exported somewhere.

Reply:
This is so we can take in anything that can be passed to moment -- thinking about component reuse. We could move this to a more shared space, but I think we usually want to specify what kind of timestamp when dealing with API responses (most of our shared types so far).